diff --git a/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js b/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js
index d466ad4..f8b3d63 100644
--- a/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js
+++ b/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js
@@ -41,7 +41,7 @@ const LinkUser = styled(RouterNavLink)`
 
   &:hover {
     background: ${({ theme, logout }) =>
-      logout ? theme.colors.danger100 : theme.colors.primary100};
+    logout ? theme.colors.danger100 : theme.colors.primary100};
     text-decoration: none;
   }
 
@@ -103,11 +103,8 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }) => {
     <MainNav condensed={condensed}>
       <NavBrand
         as={RouterNavLink}
-        workplace={formatMessage({
-          id: 'app.components.LeftMenu.navbrand.workplace',
-          defaultMessage: 'Workplace',
-        })}
-        title={menuTitle}
+        workplace={null}
+        title={null}
         icon={
           <img
             src={menu.custom || menu.default}
@@ -229,13 +226,13 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }) => {
         <NavCondense onClick={() => setCondensed((s) => !s)}>
           {condensed
             ? formatMessage({
-                id: 'app.components.LeftMenu.expand',
-                defaultMessage: 'Expand the navbar',
-              })
+              id: 'app.components.LeftMenu.expand',
+              defaultMessage: 'Expand the navbar',
+            })
             : formatMessage({
-                id: 'app.components.LeftMenu.collapse',
-                defaultMessage: 'Collapse the navbar',
-              })}
+              id: 'app.components.LeftMenu.collapse',
+              defaultMessage: 'Collapse the navbar',
+            })}
         </NavCondense>
       </NavFooter>
     </MainNav>
diff --git a/node_modules/@strapi/admin/admin/src/components/UnauthenticatedLogo/index.js b/node_modules/@strapi/admin/admin/src/components/UnauthenticatedLogo/index.js
index 0f32bdf..6b5f06d 100644
--- a/node_modules/@strapi/admin/admin/src/components/UnauthenticatedLogo/index.js
+++ b/node_modules/@strapi/admin/admin/src/components/UnauthenticatedLogo/index.js
@@ -4,6 +4,7 @@ import { useConfigurations } from '../../hooks';
 
 const Img = styled.img`
   height: ${72 / 16}rem;
+  margin-bottom: 1.5rem;
 `;
 
 const Logo = () => {
diff --git a/node_modules/@strapi/admin/admin/src/layouts/UnauthenticatedLayout/index.js b/node_modules/@strapi/admin/admin/src/layouts/UnauthenticatedLayout/index.js
index e302298..2615431 100644
--- a/node_modules/@strapi/admin/admin/src/layouts/UnauthenticatedLayout/index.js
+++ b/node_modules/@strapi/admin/admin/src/layouts/UnauthenticatedLayout/index.js
@@ -6,7 +6,7 @@ import LocaleToggle from './LocaleToggle';
 
 const Wrapper = styled(Box)`
   margin: 0 auto;
-  width: 552px;
+  width: 430px;
 `;
 
 export const Column = styled(Flex)`
diff --git a/node_modules/@strapi/admin/admin/src/pages/AuthPage/components/Login/BaseLogin.js b/node_modules/@strapi/admin/admin/src/pages/AuthPage/components/Login/BaseLogin.js
index 811a1a9..6a4c0ff 100644
--- a/node_modules/@strapi/admin/admin/src/pages/AuthPage/components/Login/BaseLogin.js
+++ b/node_modules/@strapi/admin/admin/src/pages/AuthPage/components/Login/BaseLogin.js
@@ -38,22 +38,6 @@ const Login = ({ onSubmit, schema, children }) => {
             <Form noValidate>
               <Column>
                 <Logo />
-                <Box paddingTop={6} paddingBottom={1}>
-                  <Typography variant="alpha" as="h1">
-                    {formatMessage({
-                      id: 'Auth.form.welcome.title',
-                      defaultMessage: 'Welcome!',
-                    })}
-                  </Typography>
-                </Box>
-                <Box paddingBottom={7}>
-                  <Typography variant="epsilon" textColor="neutral600">
-                    {formatMessage({
-                      id: 'Auth.form.welcome.subtitle',
-                      defaultMessage: 'Log in to your Strapi account',
-                    })}
-                  </Typography>
-                </Box>
                 {errors.errorMessage && (
                   <Typography
                     id="global-form-error"
@@ -71,9 +55,9 @@ const Login = ({ onSubmit, schema, children }) => {
                   error={
                     errors.email
                       ? formatMessage({
-                          id: errors.email,
-                          defaultMessage: 'This value is required.',
-                        })
+                        id: errors.email,
+                        defaultMessage: 'This value is required.',
+                      })
                       : ''
                   }
                   value={values.email}
@@ -90,9 +74,9 @@ const Login = ({ onSubmit, schema, children }) => {
                   error={
                     errors.password
                       ? formatMessage({
-                          id: errors.password,
-                          defaultMessage: 'This value is required.',
-                        })
+                        id: errors.password,
+                        defaultMessage: 'This value is required.',
+                      })
                       : ''
                   }
                   onChange={handleChange}
@@ -112,13 +96,13 @@ const Login = ({ onSubmit, schema, children }) => {
                       label={formatMessage(
                         passwordShown
                           ? {
-                              id: 'Auth.form.password.show-password',
-                              defaultMessage: 'Show password',
-                            }
+                            id: 'Auth.form.password.show-password',
+                            defaultMessage: 'Show password',
+                          }
                           : {
-                              id: 'Auth.form.password.hide-password',
-                              defaultMessage: 'Hide password',
-                            }
+                            id: 'Auth.form.password.hide-password',
+                            defaultMessage: 'Hide password',
+                          }
                       )}
                     >
                       {passwordShown ? <Eye /> : <EyeStriked />}
@@ -164,7 +148,7 @@ const Login = ({ onSubmit, schema, children }) => {
 
 Login.defaultProps = {
   children: null,
-  onSubmit() {},
+  onSubmit() { },
 };
 
 Login.propTypes = {
diff --git a/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js b/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js
index 1ec96b4..83b0baf 100644
--- a/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js
+++ b/node_modules/@strapi/admin/admin/src/pages/HomePage/HomeHeader.js
@@ -20,26 +20,26 @@ const HomeHeader = ({ hasCreatedContentType, onCreateCT }) => {
           <Typography as="h1" variant="alpha">
             {hasCreatedContentType
               ? formatMessage({
-                  id: 'app.components.HomePage.welcome.again',
-                  defaultMessage: 'Welcome 👋',
-                })
+                id: 'app.components.HomePage.welcome.again',
+                defaultMessage: 'Welcome 👋',
+              })
               : formatMessage({
-                  id: 'app.components.HomePage.welcome',
-                  defaultMessage: 'Welcome on board!',
-                })}
+                id: 'app.components.HomePage.welcome',
+                defaultMessage: 'Welcome on board!',
+              })}
           </Typography>
           <WordWrap textColor="neutral600" variant="epsilon">
             {hasCreatedContentType
               ? formatMessage({
-                  id: 'app.components.HomePage.welcomeBlock.content.again',
-                  defaultMessage:
-                    'We hope you are making progress on your project! Feel free to read the latest news about Strapi. We are giving our best to improve the product based on your feedback.',
-                })
+                id: 'app.components.HomePage.welcomeBlock.content.again',
+                defaultMessage:
+                  'We hope you are making progress on your project! Feel free to read the latest news about Strapi. We are giving our best to improve the product based on your feedback.',
+              })
               : formatMessage({
-                  id: 'app.components.HomePage.welcomeBlock.content',
-                  defaultMessage:
-                    'Congrats! You are logged as the first administrator. To discover the powerful features provided by Strapi, we recommend you to create your first Content type!',
-                })}
+                id: 'app.components.HomePage.welcomeBlock.content',
+                defaultMessage:
+                  'Congrats! You are logged as the first administrator. To discover the powerful features provided by Strapi, we recommend you to create your first Content type!',
+              })}
           </WordWrap>
           {hasCreatedContentType ? (
             <Link isExternal href="https://strapi.io/blog">
